builder/src/firmware.rs

```diff
@@ -152,6 +152,12 @@ pub mod driver_tests {
         features: &["emu"],
     };
 
+    const HW_LATEST_FWID: FwId = FwId {
+        crate_name: "caliptra-drivers-test-bin",
+        bin_name: "",
+        features: &["emu", "hw-latest"],
```

Comment:
Is hw-latest the right feature name for this or should this be something like hw-1.1?

Reply:
This discussion goes beyond the scope of this PR.  The current consensus which  has been approved and merged is to use  the hw-latest switch to point to the RTL release where this feature is present.